kubernetes/config/samples/imagenet.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: elastic.pytorch.org/v1alpha1
+kind: ElasticJob
+metadata:
+  name: imagenet
+  namespace: elastic-job
+spec:
+  # Use "etcd-service:2379" if you already apply etcd.yaml
+  rdzvEndpoint: "<your_etcd_endpoint>:<your_etcd_port>"
+  minReplicas: 1
+  maxReplicas: 5
+  replicaSpecs:
+    Worker:
+      replicas: 3
+      replicas: 2
+      restartPolicy: ExitCode
+      template:
+        apiVersion: v1
+        kind: Pod
+        spec:
+          containers:
+            - name: elasticjob-worker
+              image: <your_dockerhub_username>/examples:imagenet
+              imagePullPolicy: Always
+              ports:
+                - containerPort: 10291
+                  name: elasticjob-port
+              args:
+                - "s3://<your_script_path>/petctl/<username>/imagenet_job/main.py"
```

Comment:
@Jeffwan Is it expected that the s3 path is a public url? I got 403 error until I mad my script public. It will be good to include this in the troubleshooting steps. And in the instructions provide some steps for how to make the s3 url accessible to the EKS cluster -- you can point to existing AWS/EKS docs for this.

Reply:
Not necessary. Either make it public or attach S3 policy to your node group role.  I will address this part. 

The best practice is to build an example image and make sure user don't need to have additional work to run examples. We can have some refactors later or just stick to current way. 

The extra steps users need to run the example is 
1. Attach s3 policy to node group role
2. Upload training scripts to s3 
3. Update example yaml file
4. Prepare ReadWriteMany storage like EFS
5. Download dataset 

All of these steps can be skipped if we provide an example image which is built on top of `torchelastic/examples:0.1.0rc1`